core/cautils/getter/downloadreleasedpolicy_test.go

```diff
@@ -0,0 +1,130 @@
+package getter
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReleasedPolicy(t *testing.T) {
+	t.Parallel()
+
+	p := NewDownloadReleasedPolicy()
```

Comment:
This test might not pass after running many times because of the [GitHub API call limits](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#rate-limits-for-requests-from-personal-accounts).

Reply:
@dwertent  Ah right. forgot about that part. I am going to build a mock.

Reply:
@dwertent  done. The test has now the ability to download and marshal a fixture for released rego policies,
and reuse this as a mock.